app/Schedule/View.elm

```diff
@@ -198,40 +164,76 @@ laterTrainView now train =
         ]
 
 
-prediction : Date -> Train -> Node Msg
-prediction now train =
+prediction : Date -> Direction -> Train -> Node Msg
+prediction now direction train =
     let
-        predictedDeparture =
+        actualDeparture =
```

Comment:
I toyed with naming this `departure` since we don't *really* know when the actual departure will be. But then it seemed too generic. Thoughts?